wiki/Beatmap/Beatmap_tags/en.md

```diff
@@ -0,0 +1,207 @@
+# Beatmap tags
+
+*Not to be confused with [TAG difficulties](/wiki/Beatmap/TAG_beatmaps).*
+
+Keywords can be added to beatmaps to make searching for them more convenient. There are two types of tags, each with their own purpose.
+
+## Mapper tags
+
+*For rules regarding mapper tags, see: [Ranking criteria/Metadata § Tags](/wiki/Ranking_criteria/Metadata#tags)*
+
+**Mapper tags** can be used to include useful background information about a beatmap or song, e.g. the album name or the medium for which the song was produced for.
+
+When entering metadata in the [song setup window](/wiki/Client/Beatmap_editor/Song_setup), a beatmap host is able to choose custom keywords by separating them with spaces. Should a keyword itself contain spaces, they must be replaced with an underscore (`_`).
+
+The keywords can be set as a filter in [song select](/wiki/Client/Interface#song-select) as well as the [beatmap listing](https://osu.ppy.sh/beatmapsets) via full-text search.
+
+## User tags
+
+*Note: This feature is not available in osu!(stable).*
+
+**User tags** are keywords describing specific characteristics of a beatmap, such as patterns or required skill sets. After achieving at least a C [grade](/wiki/Gameplay/Grade) on a beatmap, users are able to select and vote for user tags in the result screen in osu!(lazer). Upon receiving at least 10 upvotes, the respective user tag will be displayed in the beatmap information page.
+
+To filter beatmaps containing certain user tags, click them in the beatmap information page, or type in `tag="{name}"` where `{name}` is the name of any tag listed below.
```

Comment:
Not sure how I can phrase in a succinct way that searching for user tags can be done on osu-web (beatmap listing) and lazer (integrated beatmap listing).